src/AppInstallerCommonCore/Archive.cpp

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#include "pch.h"
+
+namespace AppInstaller::Archive
+{
+    HRESULT ExtractArchive(const std::filesystem::path& archivePath, const std::filesystem::path& destPath)
+    {
+        IFileOperation* pFileOperation;
+        HRESULT hr = CoCreateInstance(CLSID_FileOperation, NULL, CLSCTX_ALL, IID_PPV_ARGS(&pFileOperation));

```

Comment:
```suggestion
        RETURN_IF_FAILED(CoCreateInstance(CLSID_FileOperation, NULL, CLSCTX_ALL, IID_PPV_ARGS(&pFileOperation)));
```
Same everywhere else.

Reply:
Changed to suggested